src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -156,6 +156,27 @@
         {% endblock messages %}
     {% endif %}
 
+    {% if collector.globalParameters is not empty %}
+        <h2>Global parameters</h2>
+
+        <table>
+            <thead>
+            <tr>
+                <th>Name</th>
+                <th>Value</th>
+            </tr>
+            </thead>
+            <tbody>
+            {% for name, value in collector.globalParameters %}
+                <tr>
+                    <td class="font-normal text-small text-bold nowrap">{{ name }}</td>
+                    <td class="font-normal text-small nowrap">{{ value }}</td>
```

Comment:
as globalParameters is an array of "string, mixed"
how will `{{ value }}` be rendered here?

Reply:
For now configuration only allows scalar values. To make it simpler we could restrict to `int|float|string| for now.

Your example seems to be related to [this comment](https://github.com/symfony/symfony/pull/53425#issuecomment-1974769159). Since a parameter can be a `TranslatableMessage`, it could be interesting to allow an other way to configure global parameters:
```yaml
# Use a value
globals:
    host: 
        value: 'https://unique.site.com' # use this value

# Use a value (short alias)
globals:
    host: 'https://unique.site.com' # use this value

# Use a message key
globals:
    host:
        message: 'host' # use TranslatableMessage('host') with the same parameters and domain

# Use a message key from an other domain
globals:
    host:
        message: 'host' # use TranslatableMessage('host') with the same parameters but "common" domain
        domain: 'common'
```

WDYT ?

Reply:
Yes can makes sens